integration/simulation/simulation.go

```diff
@@ -126,6 +126,7 @@ func (s *Simulation) Start(
 
 	// todo - changing from time to common will delay the node start and it will not catch the first few blocks
 	s.l1Network.Start(time.Duration(s.avgBlockDuration / 4))
+	time.Sleep(1 * time.Second)
```

Comment:
Can we add a comment explaining what the sleep here introduces, think it will be helpful

Reply:
good point!